src/libstd/macros.rs

```diff
@@ -213,6 +213,26 @@ macro_rules! eprintln {
     ($fmt:expr, $($arg:tt)*) => (eprint!(concat!($fmt, "\n"), $($arg)*));
 }
 
+#[macro_export]
+#[unstable(feature = "await_macro", issue = "50547")]
+#[allow_internal_unstable]
```

Comment:
This should probably use `#[allow_internal_unsafe]` as well.

Reply:
Done.